### PR TITLE
Landing page design has to be changing according to the selected platform

### PR DESCRIPTION
**Pull request by valens10**
#### What does this PR do?
Enable changing of landing page interface according to the selected platform
#### Description of Task to be completed?
- Create main function and add login page(Landing page)
- Add script to check which platform has selected
#### How should this be manually tested?
- Clone the repo git clone `https://github.com/valens10/flipper-core.git`
- Switch your working directory `new-design`
- Checkout the branch `git checkout new-design`
- run the app
#### What are the relevant Issues stories?
[#2](https://github.com/yegobox/flipper-core/issues/2)

#### Screenshots?
Screen Shot 2021-03-11 at 17 00 09

Screen Shot 2021-03-11 at 17 02 07

### CODE REVIEW

File: packages/flipper_login/lib/login_view.dart [hunk 1]
```diff
@@ -0,0 +1,45 @@
...
+class LoginView extends StatefulWidget {
+  const LoginView({Key? key}) : super(key: key);
+
+  @override
+  LoginViewState createState() => LoginViewState();
+}
+
+class LoginViewState extends State<LoginView> {
+  @override
+  Widget build(BuildContext context) {
```
Comment (richard457) [inline packages/flipper_login/lib/login_view.dart:15]: I like the use of the library to know the platform which we are on but it is misused for this case, it should be used for a case where we want to decide functionality that is on one platform not other, so basically for eg: all app should share the same code i.e one Login but differ to small details.

**Review by richard457 — CHANGES_REQUESTED** [review]
The PR should address the issue (Feature described here [issue](https://github.com/yegobox/flipper-core/issues/2) not only to differentiate the platform

File: packages/flipper_login/lib/login_view.dart (hunk 1 above)
Comment (richard457) [reply packages/flipper_login/lib/login_view.dart:15]: so we should not have 4 different UI

Comment (valens10) [reply packages/flipper_login/lib/login_view.dart:15]: The PR is still in-progress and need the suggestions

Comment (richard457) [reply packages/flipper_login/lib/login_view.dart:15]: To put it into simple explanation we should have one LoginView (Widget) that is responsive to all platform we have given to different screen size you can learn more about responsiveness  [Here](https://github.com/yegobox/flutter_facebook_responsive_ui)

Comment (valens10) [reply packages/flipper_login/lib/login_view.dart:15]: Got it

**Pull request merged**